tests/bulk/bulkwrite-update_error-007.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+MongoDB\Driver\BulkWrite::update() arrayFilters option requires MongoDB 3.6
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_server_version('>=', '3.6'); ?>
```

Comment:
Note: since we're testing for client-side errors, we don't actually care about whether the collection is clean or not. We also don't bother inserting any data before running the `update()` operation.